inject.go

```diff
@@ -69,37 +87,67 @@ func Inject(target interface{}) Option {
 	// 		target.Field(2),  // Baz io.Writer
 	// 	]
 	//
-	// As we iterate through the arguments received by the function, we can
-	// simply copy the value into the corresponding value in the targets list.
+	// As we iterate through the fields of the generated struct, we can simply
```

Comment:
No need to put "simply". Suggest: "Iterate through the fields of the generated struct and copy the value..."